Spigot-Server-Patches/0646-Implement-Keyed-on-World.patch

```diff
@@ -0,0 +1,100 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Jake Potrebic <jake.m.potrebic@gmail.com>
+Date: Wed, 6 Jan 2021 00:34:04 -0800
+Subject: [PATCH] Implement Keyed on World
+
+
+diff --git a/src/main/java/net/minecraft/server/IRegistry.java b/src/main/java/net/minecraft/server/IRegistry.java
+index 6c7caef36b484a16b62977b5495247588c34dc2b..accdc9e040e1aa816835dd1ce157cfcabb39cb93 100644
+--- a/src/main/java/net/minecraft/server/IRegistry.java
++++ b/src/main/java/net/minecraft/server/IRegistry.java
+@@ -57,7 +57,7 @@ public abstract class IRegistry<T> implements Codec<T>, Keyable, Registry<T> {
+     public static final ResourceKey<IRegistry<LootItemFunctionType>> I = a("loot_function_type");
+     public static final ResourceKey<IRegistry<LootItemConditionType>> J = a("loot_condition_type");
+     public static final ResourceKey<IRegistry<DimensionManager>> K = a("dimension_type");
+-    public static final ResourceKey<IRegistry<World>> L = a("dimension");
++    public static final ResourceKey<IRegistry<World>> L = a("dimension"); public static final ResourceKey<IRegistry<World>> getDimensionRegistry() { return L; } // Paper - OBFHELPER
```

Comment:
This is a bit of a funky encounter of Mojang vs. community names, their "dimension" is our "world" (making this a worldRegistry)

Reply:
and I thought their level was our world :P

obfhelpers dont matter anyways, and if they use the vanilla/mojang name, they are easier to remove when mojmap is done

Reply:
Weeell, they do as well, see `DIMENSION_REGISTRY = Registry.<Level>createRegistryKey("dimension")` 😂